src/table/utils/handle-accessibility.ts

```diff
@@ -28,23 +48,43 @@ export const updateFocus = ({ focusType, cell }) => {
   }
 };
 
-export const removeAndFocus = (newCoord, rootElement, setFocusedCellCoord, keyboard) => {
+export const removeAndFocus = (
+  newCoord: [number, number],
+  rootElement: Element,
+  setFocusedCellCoord: (coord: [number, number]) => void,
+  keyboard: stardust.Keyboard
+) => {
   updateFocus({ focusType: 'removeTab', cell: findCellWithTabStop(rootElement) });
   setFocusedCellCoord(newCoord);
-  keyboard.enabled && keyboard.focus();
+  keyboard.enabled && keyboard.focus?.();
 };
 
-export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard, totalsPosition) => {
+export const handleClickToFocusBody = (
+  cell: { rawRowIdx: number; rawColIdx: number },
+  rootElement: Element,
+  setFocusedCellCoord: (coord: [number, number]) => void,
+  keyboard: stardust.Keyboard,
+  totalsPosition: string
```

Comment:
There is a type for this in types. It limits it to three different values, not just any string